pkg/monitor/controller.go

```diff
@@ -365,3 +379,14 @@ func (monitor *controllerMonitor) deleteAgentCRD(name string) error {
 	}
 	return nil
 }
+
+func (monitor *controllerMonitor) AgentInfoCRDInstalled() bool {
```

Comment:
`clientset.Interface` has `Discovery()` which can be used to check if an API exists. We already use it to check EndpointSlice API. This is an example: https://github.com/antrea-io/antrea/blob/f1545c176767a94ba0aed470d3886a9c7a1bf1cc/pkg/agent/proxy/proxier.go#L1158-L1173

Reply:
updated.

Reply:
it should be a private method, and something like `antreaAgentInfoAPIAvailable` reflects the implementation more accurately.

Reply:
reminder

Reply:
updated.